announcement/announcement-tool/tool/src/java/org/sakaiproject/announcement/entityprovider/AnnouncementEntityProviderImpl.java

```diff
@@ -130,6 +135,7 @@ public String getEntityPrefix() {
 		//we use this zero value to determine if we need to look up from the tool config, or use the defaults if still not set.
 		int numberOfAnnouncements = NumberUtils.toInt((String)params.get("n"), 0);
 		int numberOfDaysInThePast = NumberUtils.toInt((String)params.get("d"), 0);
+		boolean announcementSortAsc = NumberUtils.toInt((String)params.get("a"), 0) == 1 ? true:false;
```

Comment:
BooleanUtils maybe more appropriate here

Reply:
Do you mean something like this:

`boolean announcementSortAsc = BooleanUtils.toBoolean(NumberUtils.toInt((String)params.get("a"), 0));`

However, the issue I have with that is values greater than 1 will also evaluate to true.  I looked at other methods in BooleanUtils, but I couldn't find anything else that didn't require catching exceptions or checking for null afterwards.